opus/application/static_media/js/opus.js

```diff
@@ -336,11 +335,11 @@ var opus = {
         opus.widgets_drawn = [];
```

Comment:
I'm confused by this line. Why are we setting widgets to [] instead of to the default_widgets?

Reply:
Here is what we do in startOver:
1. We empty the widget area first (opus.js line 323). 
2. Later on when we are looping through opus.default_widgets and calling o_widgets.getWidget for each item in opus.default_widgets (opus.js line 354), we will update opus.widgets_drawn in o_widgets.getWidget (widgets.js line 559). 

So If we set opus.widgets_drawn to opus.default_widgets here, in o_widgets.getWidget, it will think those widgets are already drawn and return from the function (widgets.js line 374). That way those widgets won't be drawn, and widget area will still be empty.